src/components/Person/Person.jsx

```diff
@@ -1 +1,16 @@
-// export const Person = ({ person }) => ();
+export const Person = ({ person }) => {
+  const { name, age, sex, isMarried, partnerName } = person;
+  const partnerTitle = sex === 'm' ? 'wife' : 'husband';
+
+  return (
+    <section className="Person">
+      <h2 className="Person__name">My name is {name}</h2>
+      {age && <p className="Person__age">I am {age}</p>}
+      <p className="Person__partner">
+        {isMarried
+          ? `${partnerName} is my ${partnerTitle}`
+          : 'I am not married'}
+      </p>
```

Comment:
Fix indents and ur ternary syntax for better readability 
```suggestion
      <h2 className="Person__name">My name is {name}</h2>
      
      {age && <p className="Person__age">I am {age}</p>}
      
      <p className="Person__partner">
        {
          isMarried
            ? `${partnerName} is my ${partnerTitle}`
            : 'I am not married'
        }
      </p>
```